diga-epa-client/src/test/java/com/oviva/epa/client/KonnektorServiceAcceptanceTest.java

```diff
@@ -239,13 +242,13 @@ private Document buildDocumentPayload(
             "application/fhir+xml",
             PracticeSettingCode.PATIENT_AUSSERHALB_BETREUUNG.getValue(),
             List.of(),
-            LocalDateTime.now().minusWeeks(2),
-            LocalDateTime.now().minusWeeks(1),
+            null,
+            null,
```

Comment:
why did you leave them out? we don't have enough data for this?

Reply:
Because we don't have reasonable values as of yet. We would need to extract the start date of treatment in CORE and forward that to the epa-exporter and the diga-epa-lib.